package.json

```diff
@@ -31,7 +31,6 @@
     "clean": "gulp clean",
     "clean:build": "gulp cleanBuildDir",
     "clean:release": "gulp cleanReleaseDir",
-    "checkin": "gulp checkin",
```

Comment:
Could we remove some more of these? e.g. now that everything properly calls prereqs, is there any reqson to call `clean:build` manually?